### ISSUE TITLE
TimeoutError: waiting for selector "[aria-label="Microphone"]" failed: timeout 30000ms exceeded

### ISSUE THREAD

**Issue by rexon07** [openfun/bbb-stress-test#10]
Hi, I used this script , it works well up to 17 users from 18th user I get the timeout error,  for which I edited the stress-test.js and set the {timeout : 0 }  after which the error does not appear but on the 18th user joining it gets stuck at : https://prnt.sc/10gyvck

Looking for help Thanks.

**Comment by madmatah** [comment]
Hi @rexon07,

Maybe the machine on which you ran the `bbb-stress-test` script is overloaded?
It's quite resource-intensive and I used to run it on multiple VMs.

**Comment by rexon07** [comment]
Hi @madmatah 

Thanks for your reply , I will try that out:)

**Comment by rexon07** [comment]
Hi @madmatah 

I tested  it on a 8core , 32gb  result is same as the previous one , earlier I ran the script on 4core 8gb server with other apps running.
server result : https://prnt.sc/10jiyd3
max id adds upto 17users and on the 18th user it gets stuck here : https://prnt.sc/10jj000

**Comment by besauer** [comment]
I have the same problem.

@madmatah could it be a problem with bbb 2.3? With bbb 2.2 the stress test works propper.

**Comment by madmatah** [comment]
Hi @besauer 
For me, this issue is related to a resource problem (load average > 30 in the previous screenshots).
The stress test should be launched from several machines.

Unfornately, I haven't tested it on bbb 2.3 and currently I have no such environment to test it. Maybe there is something problematic with this version, yes.

**Comment by jonnyfux** [comment]
It is the same for me, we are running bbb 2.2 and even using a c5.9xlarge with 36 cores resulted in the error after 17 users with webcam and audio....

Listen only works up to around 100

**Comment by olwe0002** [comment]
for me it was the same, starting from the first joined user, because I had `autoShareWebcam` enabled in my BBB instance. To make it possible, I added an additionally option, see my MR [here](https://github.com/openfun/bbb-stress-test/pull/16) or my [fork](https://github.com/olwe0002/bbb-stress-test)
